### ISSUE TITLE
Support for `explicitHashKey`

### ISSUE THREAD

**Issue by nkijak** [MyPureCloud/mock-kinesis-maven-plugin#2]
When working with `com.amazonaws.kinesis.agg.ArrRecord`, it will set `explicitHashKey` which will then error out on the mock stream due to Jackson rejecting the unknown property.

```
com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "ExplicitHashKey" (class com.genesys.purecloud.request.PutRecordRequest), not marked as ignorable (3 known properties: "Data", "StreamName", "PartitionKey"])
```

**Comment by KevinGlinski** [comment]
i'll take a look tomorrow

**Comment by KevinGlinski** [comment]
fixed in 1.1.1
